src/mod/server.mod/servmsg.c

```diff
@@ -2205,7 +2205,7 @@ static void server_resolve_failure(int);
  */
 static void connect_server(void)
 {
-  char pass[121], botserver[UHOSTLEN], s[1024];
+  char pass[sizeof newserverpass], botserver[sizeof newserver], s[1024];
```

Comment:
prefer the defines here, newserverpass could be `extern char *` or `extern char[]` and then sizeof would be wrong

Reply:
Done.